Commit ID: df74722c1bf2e02c17a8a5592cd9038c24ec699a
Change ID: qvvpznxopmzmvvsuznzmkromuvxqrpvq
Author   : Lauri Tirkkonen <lauri@hacktheplanet.fi> (2025-11-03 14:11:03)
Committer: Lauri Tirkkonen <lauri@hacktheplanet.fi> (2025-11-03 14:11:06)

    test_multiple: double test time

    this test times out on the ppc64le builder, despite being fine in CI and
    succeeding previously on the builder when this package was in testing.

diff --git a/horust/tests/horust.rs b/horust/tests/horust.rs
index 950c616695..2efee0f352 100644
--- a/horust/tests/horust.rs
+++ b/horust/tests/horust.rs
@@ -91,7 +91,7 @@
     }
     store_service_script(temp_dir.path(), script, None, Some("0"));
     let recv = run_async(&mut cmd, true);
-    recv.recv_or_kill(Duration::from_secs(max * 2));
+    recv.recv_or_kill(Duration::from_secs(max * 4));
 }
 
 #[test]
